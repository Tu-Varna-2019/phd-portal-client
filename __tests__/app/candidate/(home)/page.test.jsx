import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Page from "../../../../src/app/candidate/(home)/page.jsx";

describe("Page", () => {
  it("renders a heading", () => {
    render(<Page />);

    const progressbar = screen.getByRole("progressbar");

    expect(progressbar).toBeInTheDocument();
  });
});

it("renders homepage unchanged", () => {
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
