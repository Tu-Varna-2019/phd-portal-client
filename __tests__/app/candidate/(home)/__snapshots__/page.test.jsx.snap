// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="MuiBox-root css-101ksu7"
  >
    <span
      class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-vkmcds-MuiCircularProgress-root"
      role="progressbar"
      style="width: 80px; height: 80px;"
    >
      <svg
        class="MuiCircularProgress-svg css-54pwck-MuiCircularProgress-svg"
        viewBox="22 22 44 44"
      >
        <circle
          class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-19t5dcl-MuiCircularProgress-circle"
          cx="44"
          cy="44"
          fill="none"
          r="20.2"
          stroke-width="3.6"
        />
      </svg>
    </span>
  </div>
</div>
`;
